description: check that exit status is 2

--- a/client/dhclient.c
+++ b/client/dhclient.c
@@ -1256,7 +1256,7 @@ void bind_lease (client)
 	/* If the BOUND/RENEW code detects another machine using the
 	   offered address, it exits nonzero.  We need to send a
 	   DHCPDECLINE and toss the lease. */
-	if (script_go(client)) {
+	if (script_go(client) == 2) {
 		make_decline(client, client->new);
 		send_decline(client);
 		destroy_client_lease(client->new);
