description: loopback interface isn't checksumed

--- a/common/packet.c
+++ b/common/packet.c
@@ -336,6 +336,11 @@ decode_udp_ip_header(struct interface_in
 				  checksum((unsigned char *)&ip.ip_src,
 					   8, IPPROTO_UDP + ulen))));
 
+  /* loopback interface isn't checksumed in linux */
+  if (interface -> hw_address.hbuf [0] == 0) {
+    usum = 0;
+  }
+
   udp_packets_seen++;
   if (usum && usum != sum) {
 	  udp_packets_bad_checksum++;
